tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,46 @@
+package cdi_test
+
+import (
+	"flag"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Importing and starting a VM using CDI", func() {
+	flag.Parse()
+	ktests.SkipIfNoOc()
+	dstPVCFilePath := "/tmp/test-pvc.json"
+	dstVMFilePath := "/tmp/test-vm.json"
+
+	url, ok := os.LookupEnv("STREAM_IMAGE_URL")
+	if !ok {
+		url = pvcEPHTTPNOAUTHURL
+	}
+
+	Context("PVC with valid image url will succeed", func() {
+		It("should create the PVC successfully", func() {
+			tests.ProcessTemplateWithParameters(rawPVCFilePath, dstPVCFilePath, "PVC_NAME="+pvcName, "EP_URL="+url)
+			tests.CreateResourceWithFilePathTestNamespace("pvc", pvcName, dstPVCFilePath)
+		})
+
+		Specify("the PVC should become bound", func() {
+			tests.WaitUntilResourceReadyByNameTestNamespace("pvc", pvcName, "-o=jsonpath='{.metadata.annotations}'", "pv.kubernetes.io/bind-completed:yes pv.kubernetes.io/bound-by-controller:yes")
```

Comment:
Since annotations are maps I don't think you can rely on the order of 2 annotations. It is more reliable and accurate to test each annotation value separately, imo.

Reply:
I guess I can only keep one, `pv.kubernetes.io/bind-completed:yes` is already enough.